<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobTableWidget</class>
 <widget class="QWidget" name="JobTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="MoleQueue::JobView" name="table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="filterBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filter jobs from the table.&lt;/p&gt;&lt;p&gt;Each whitespace separated term will be used to restrict which jobs are displayed in the job table. Entries in the table that match terms here will be displayed. Terms beginning with '-' will negate the match.&lt;/p&gt;&lt;p&gt;For example, the filter &amp;quot;water -opt&amp;quot; will match a job with the title &amp;quot;Water single point&amp;quot;, but not &amp;quot;Water geometry optimization&amp;quot;. Matches are made against all visible text in the job entry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="edit-delete">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="filterMore">
            <property name="text">
             <string>More...</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoleQueue::JobView</class>
   <extends>QTableView</extends>
   <header>jobview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterEdit</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>filterMore</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>filterEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
